topostats/tracing/tracingfuncs.py

```diff
@@ -816,6 +227,9 @@ def loopedLinearTrace():
 class genTracingFuncs:
```

Comment:
Don't really need this to be a class if its only got a series `@staticmethod` it can just be a series of functions.

Might perhaps be better to refactor as...

```
class genTracingFuncs():
    """Class for generating tracing functions."""
    def __init__(self, binary_map: np.ndarray, x: int, y:int) -> None:
    """Initialise the class."""
    self.binary_map = binary_map
    self.x = x
    self.y = y

    ...
```

Reply:
I'd first say that I like how the class groups these functions and thus enhances readability. I also don't think your suggested refactor would work as some of the functions take points etc.

This might be one for when this entire part (the rest of dnatracing) has been refactored